tests/e2e/performance/csi_tests/test_pvc_bulk_clone_performance.py

```diff
@@ -1,27 +1,101 @@
 """
 Test to measure pvc scale creation time. Total pvc count would be 50, 1 clone per PVC
 Total number of clones in bulk will be 50
+The results are uploaded to the ES server
 """
 import logging
 import pytest
+from uuid import uuid4
 
 from ocs_ci.utility import utils
 from ocs_ci.ocs.perftests import PASTest
 from ocs_ci.framework.testlib import performance
+from ocs_ci.framework import config
 from ocs_ci.helpers import helpers, performance_lib
+from ocs_ci.helpers.helpers import get_full_test_logs_path
 from ocs_ci.ocs import constants, scale_lib
 from ocs_ci.ocs.resources import pvc, pod
 from ocs_ci.ocs.resources.objectconfigfile import ObjectConfFile
+from ocs_ci.ocs.perfresult import PerfResult
 
 log = logging.getLogger(__name__)
 
 
+class ResultsAnalyse(PerfResult):
+    """
+    This class generates results for all tests as one unit
+    and saves them to an elastic search server on the cluster
+
+    """
+
+    def __init__(self, uuid, crd, full_log_path):
+        """
+        Initialize the object by reading some of the data from the CRD file and
+        by connecting to the ES server and read all results from it.
+
+        Args:
+            uuid (str): the unique uid of the test
+            crd (dict): dictionary with test parameters - the test yaml file
+                        that modify it in the test itself.
+            full_log_path (str): the path of the results files to be found
+
+        """
+        super(ResultsAnalyse, self).__init__(uuid, crd)
+        self.new_index = "bulk_clone_perf_fullres"
+        self.full_log_path = full_log_path
+        # make sure we have connection to the elastic search server
+        self.es_connect()
+
+
 @performance
 class TestBulkCloneCreation(PASTest):
     """
     Base class for bulk creation of PVC clones
     """
 
+    def setup(self):
+        """
+        Setting up test parameters
+        """
+        logging.info("Starting the test setup")
+        super(TestBulkCloneCreation, self).setup()
+        self.benchmark_name = "pod_bulk_clone_creation_time"
+        self.uuid = uuid4().hex
+        self.crd_data = {
+            "spec": {
+                "test_user": "Homer simpson",
+                "clustername": "test_cluster",
```

Comment:
Would it be possible to change the name of the cluster to a unique name? (maybe as a subsequent PR)

Reply:
@keesturam 
I need to check whether it is possible to get this unique name from the cluster. 
Generating a unique random string is not good since wwe want the same unique name for all the tests run on the same cluster.

Reply:
I agree that it cannot be a randomly generated name. 
maybe from the env variables, we can get the clustername+ocpversion+odfversion?